### DISCUSSION TITLE
A real world app for demo

### DISCUSSION THREAD

**Discussion by frederikhors** [jycouet/kitql#72] [Ideas]
I think it would be great to create a https://github.com/gothinkster/realworld app with SvelteKit.

**Comment by jycouet** [comment]
Cool!
I didn't know that something like this exist!
They speak a lot about a REST API, can we do graphql? Can we do a graphql server?

_Too bad that they are using `Bootstrap` for the UI ^^_

**Reply by jycouet** [reply]
They don't seem to be that strict, you can do your own graphql and still submit a "Realworld" app :)
Do you know if alternatives exist?

**Reply by frederikhors** [reply]
Nope. But I have seen that there are projects around with the most disparate technologies.

**Comment by jycouet** [comment]
When do we start the hackathon? 😉
Who is in?

**Reply by frederikhors** [reply]
I will create it tomorrow but only for having a complex demo for our goals.

**Reply by frederikhors** [reply]
Can you please focus on partial results? LOL! 😄

**Reply by jycouet** [reply]
I was more thinking "hackathon" in the sense of: we put a start date, leave 24h and see the result of each people ^^

But if you wanna start, do not hesitate. It's a great way to learn.

**Reply by frederikhors** [reply]
Yeah. I'm not an hackathon person.

**Comment by spences10** [comment]
I'm building an example with [Vendure](https://www.vendure.io)

Source: https://github.com/spences10/sveltekit-vendure-commerce

Not sure if that fits with what you're discussing here though

**Reply by jycouet** [reply]
Another one with Supabase will be merged soon I think 🥳
